src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -8,7 +8,28 @@
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws CustomeException {
+        if (user.getAge() == null || user.getPassword() == null || user.getLogin() == null) {
+            throw new CustomeException("User login,password or age is null!");
+        }
+        if (user.getLogin().isEmpty() || (user.getPassword().isEmpty())) {
+            throw new CustomeException("User login or password is empty!");
```

Comment:
The exception message could be more specific. Consider specifying which field is empty instead of a generic message.